sacred/commands.py

```diff
@@ -183,7 +183,17 @@ def _format_entry(indent, entry):
         doc_string = entry.value.replace('\n', '\n' + indent)
         assign = '{}"""{}"""'.format(indent, doc_string)
     elif isinstance(entry, ConfigEntry):
-        assign = indent + entry.key + " = " + PRINTER.pformat(entry.value)
+        pre = indent + entry.key + " = "
+        entry_str = PRINTER.pformat(entry.value)
+
+        def add_multi_line_indent(entry_str, indent):
```

Comment:
Wouldn't a simple `entry_str.replace('\n', '\n' + indent)` accomplish the same?

Reply:
Revisiting my code now it seems that the recursion is very much over engineered. Yes, your approach should be sufficient. But I would not just add the indent but the key and plus sign:
`entry_str.replace('\n', '\n' + ' ' * len(pre))`

Reply:
Sounds good to me. If you adjust this line I'd be happy to merge.